services/QuillLMS/db/migrate/20240620123600_add_various_fields_to_trial_and_llm_prompt.evidence.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# This migration comes from evidence (originally 20240620122344)
+class AddVariousFieldsToTrialAndLLMPrompt < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :evidence_research_gen_ai_trials, :num_examples, :integer
+    rename_column :evidence_research_gen_ai_trials, :stem_vault_id, :dataset_id
```

Comment:
This a hack because stem_vault is a different object than dataset.  This change is about injecting a new parent 
relationship.

StemVault -> Trial becomes StemVault -> Dataset -> Trial